wallet/lib/pages/detailed_asset_view/widgets/nft_image_asset.dart

```diff
@@ -35,7 +35,7 @@ class NftImageWidget extends StatelessWidget {
                   BlendMode.srcOver,
                 ),
                 image: imageProvider,
-                fit: BoxFit.fitHeight,
```

Comment:
Why we changed this?

Reply:
As per the requirement, we need to show the full image to the user when NFT is viewed, so that is why we changed it to fitWidth instead of fitHeight to show user full image